datetime_formatter/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Datetime Formatter
+==================
+
+This module was written to extend the functionality of Odoo language engine to
+support formatting `Date`, `Time` and `Datetime` fields easily and allow you to
+print them in the best format for the user.
+
+Usage
+=====
+
+This module adds a technical programming feature, and it should be used by
+addon developers, not by end users. This means that you must not expect to see
+any changes if you are a user and install this, but if you find you have it
+already installed, it's probably because you have another modules that depend
+on this one.
+
+If you are a developer, to use this module, you need to:
+
+* Call anywhere in your code::
+
+    formatted_string = self.env["res.lang"].datetime_formatter(datetime_value)
+
+* If you use Qweb::
+
+    <t t-esc="env['res.lang'].datetime_formatter(datetime_value)"/>
```

Comment:
@pedrobaeza This is the main andvantage. The method resides in a DB model, so it can be called from wherever the registry is available, as in Qweb views. You cannot import from other modules in Qweb unless you create a custom controller for that view. This is much simpler.